db/migrate/20170818154724_create_resource_contents.rb

```diff
@@ -0,0 +1,11 @@
+class CreateResourceContents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :resource_contents do |t|
+      t.string :multihash, null: false
+      t.text :body, null: false
```

Comment:
Aren't all our supported metadata formats text based?  Why would you consider binary? Are there any non-printing characters we mean to support?